sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Config/EventHubOptions.cs

```diff
@@ -132,6 +134,52 @@ public int MaxEventBatchSize
             }
         }
 
+        private int _minEventBatchSize;
+
+        /// <summary>
+        /// Gets or sets the minimum number of events delivered in a batch. This setting applies only to functions that
+        /// receive multiple events. This value must be less than <see cref="MaxEventBatchSize"/>. This value is used in
+        /// conjunction with <see cref="MaxWaitTime"/>. Default 1.
+        /// </summary>
+        public int MinEventBatchSize
+        {
+            get => _minEventBatchSize;
+
+            set
+            {
+                if (value < 1)
+                {
+                    throw new ArgumentException("Batch size must be larger than or equal to 0.");
+                }
+                _minEventBatchSize = value;
+            }
+        }
+
+        private TimeSpan _maxWaitTime;
+
+        /// <summary>
+        /// Gets or sets the maximum time in seconds. This only applies when <see cref="MinEventBatchSize"/> is set. It denotes the
+        /// maximum number of seconds the processor will wait after receiving a batch of events less than <see cref="MinEventBatchSize"/>
+        /// before invoking the function. Default is 60 seconds when <see cref="MinEventBatchSize"/> has been set, otherwise, this is not used.
```

Comment:
```suggestion
        /// Gets or sets the maximum time that the trigger should wait to fill a batch before invoking the function.  This is only considered when <see cref="MinEventBatchSize"/> is set to larger than 1 and is otherwise unused.  If less than <see cref=MinEventBatchSize" /> events were available before the wait time elapses, the function will be invoked with a partial batch.  Default is 60 seconds.  The maximum wait time is 10 minutes.
```